### ISSUE TITLE
Configs to reduce merge noise

### ISSUE THREAD

**Issue by jnorthrup** [automenta/narchy#15]
i noticed quite a few constants are tweaked and committed which looks like a reasonable place to plow in some configfiles with defaults.

I'm supremely happy with a particular incarnation i have from blockchain and production works.

**Comment by automenta** [comment]
yes its a good feature to add but i had some ideas about doing that.  im connected on frenode /query me

**Comment by automenta** [comment]
some of the statics in Param are there only temporarily until being made into dynamic runtime modifiable parameters and others are actually static and should be in config files rather than in .java.  ill have to sort these out carefully.

i imagined that it should be possible to make the config files as similar to editing .java code as possible, or NAL or both -- something JSON-like, etc.  i'm for anything that doesnt introduce yet another syntax and makes use of an existing one.  JDK12+ has some kind of feature for running .java files like a scripting language. 
there's also the runtime DI-like configuration that is emerging as can be seen in GameX.java - the various components that can be wired into the container etc should be just as configurable as the Param stuff.  but here is where the static and runtime configuration gets blurry because whatever is runtime configurable is meant to be tweakable by both the system itself AND the GUI - or both as you see the knobs can move by themselves but you can also fight with them (these will need manual override buttons too)

**Comment by jnorthrup** [comment]
see sparkly
